.github/workflows/linux.yml

```diff
@@ -124,6 +124,9 @@ jobs:
         python dev.py build --tags=runtime,python-runtime,devel,tests
         python tools/check_installation.py ${{ env.INSTALLDIR }}
 
+    - name: Check build-internal dependencies
+      run: ninja -C build -t missingdeps
```

Comment:
This is really cool (could have saved me a bisection a few weeks ago) and I checked locally that indeed it goes from exit code `3` to `0` with your patch. I think we need a `meson` version bump to get the CI passing in some places.

Reply:
Yes indeed. That should be okay, and the thing that broke in CI is a quite useful change in Meson that allowed some more cleanups.